grid.css

```diff
@@ -10,57 +10,79 @@ heading1, heading2{
 /* style for the headings section */
 heading1{
 	padding: 0;
-    border-bottom: 1px solid white;
+    border-bottom: 0.2vw solid #ffffff;
 	grid-template-rows: 100%;
 	background-color: #292a33;	
   	text-align: center;
 	position: sticky;
 	top: 0vw;
+    box-shadow: 0 10px 6px -6px #18191f;
 }
 
-/* style for the headings song details section */
+Song_ID1, Title1, Artist1, Album1, Genre1, Duration1, Size1{
+	width: 1.5vw;
+}
+
+/* style for the song output section */
 heading2{
 	grid-template-rows: 100%;
+	border-bottom: 0.05vw solid #ffffff;
```

Comment:
#added bottom border to each song output to help each of them stand out from each other